### PR TITLE
Add Veepee icon

### PR DESCRIPTION
**Pull request by KarthikNayak024**
**Issue:** #3462 
**Alexa rank:** 6,136 
  

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json`
  - [ ] I optimized the icon with SVGO or SVGOMG
  - [ ] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6697,9 +6697,14 @@
...
             "title": "Veeam",
-            "hex": "00B336",
+            "hex": "FF008E",
             "source": "https://www.veeam.com/newsroom/veeam-graphics.html"
         },
+        {
+            "title": "Veepee",
+            "hex": "00B336",
+            "source": "http://www.veepee.com/"
+        },
```
Comment (service-paradis) [inline _data/simple-icons.json:6707]: I think you have changed the color at the wrong place in `simple-icons.json`.
Also, the color used in the logo and in the website is `#EC008C` (https://static.preprod.vente-privee.com/vp5/_web/site/vp/_common/img/00_int_door_logoBlack.svg).

**Review by service-paradis — CHANGES_REQUESTED** [review]
Thanks for your contribution 😃 
Maybe they updated their logo, but it looks different when I compared your version with the one found in the website header:
image
Can you take a look?

**Comment by adamrusted** [comment]
veepee

Rebuilt from source using @service-paradis' suggestions.

**Comment by service-paradis** [comment]
LGTM 🦋 🎉 
Thanks @KarthikNayak024 and @adamrusted!

**Pull request merged**